docs/security.md

```diff
@@ -47,6 +48,17 @@ follows:
 * Import all exported public keys into a single trust-store
 * Distribute the trust-store over the nodes
 
+### Configuring SASL Encryption
+
+To enable SASL encryption for an application, set `spark.authenticate.enableSaslEncryption` to
```

Comment:
Can you add one sentence to explain what benefits SASL encryption provides (e.g. clarify where it applies)?